plugin/src/main/java/org/opensearch/ml/utils/error/ErrorMessage.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.utils.error;
+
+import org.json.JSONObject;
```

Comment:
I am not sure whether we should leverage `ToXContentObject` interface which provided by OpenSearch to do json/yml convert and try to not include a new lib.

Reply:
I just tried to use ToXContentObject but the related methods would throw exception again, so I want to keep this raw json handling.

Reply:
As Sarat suggested, I removed json library dependency and used ObjectMapper.